pkg/sql/sem/tree/name_resolution_test.go

```diff
@@ -750,7 +750,7 @@ func TestResolveTablePatternOrName(t *testing.T) {
 		// Names of length 3
 
 		{`db3.public.foo`, `db3`, tpath("pg_temp_123", "public"), true, `db3.public.foo`, `db3.public.foo`, `db3.public[0]`, ``},
-		{`db3.pg_temp.foo`, `db3`, tpath("pg_temp_123", "public"), true, `db3.pg_temp.foo`, `db3.pg_temp.foo`, `db3.pg_temp[0]`, ``},
+		{`db3.pg_temp.foo`, `db3`, tpath("pg_temp_123", "public"), true, `db3.pg_temp_123.foo`, `db3.pg_temp_123.foo`, `db3.pg_temp_123[0]`, ``},
```

Comment:
does the case for `pg_temp` not work anymore?

Reply:
all of the `pg_temp`resolution still works, we just return the resolved schema name here rather than keeping it as `pg_temp` -- postgres does this as well.